src/ucp/tag/tag_send.c

```diff
@@ -166,6 +167,18 @@ ucp_tag_send_inline(ucp_ep_h ep, const void *buffer, size_t length, ucp_tag_t ta
 
     if (status != UCS_ERR_NO_RESOURCE) {
         UCP_EP_STAT_TAG_OP(ep, EAGER);
+        #if ENABLE_DEBUG_DATA
+            if (ucs_unlikely(param->op_attr_mask & UCP_OP_ATTR_FIELD_DEBUG_INFO)) {
+                UCS_STRING_BUFFER_FIXED(strb, param->debug_info->debug_string,
+                                        param->debug_info->debug_string_size);
+                char lane_info[128] = {0};
```

Comment:
better define it under `#if ENABLE_DEBUG_DATA` at the begging of the function as it is required by code style
thanks!